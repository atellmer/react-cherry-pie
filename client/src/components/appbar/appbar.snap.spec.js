import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

import { windowMatchMedia } from '../../../test/__mocks__/matchMediaMock';
import muiTheme from '../../config/theme';
import TmAppbar from './index';

jest.mock('react-dom');
jest.mock('react/lib/ReactDefaultInjection');

const me = {
  id: '',
  name: {
    first: '',
    last: ''
  },
  avatar: {
    thumbnail: ''
  },
  online: false
};

beforeEach(() => {
  window.matchMedia = windowMatchMedia;
});

test('Jest: TmAppbar (Snapshot)', () => {
  const component = renderer.create(
    <MuiThemeProvider muiTheme={muiTheme}>
      <TmAppbar me={me}/>
    </MuiThemeProvider>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
