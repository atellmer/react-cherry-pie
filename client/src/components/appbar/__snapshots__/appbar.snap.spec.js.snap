exports[`test Jest: TmAppbar (Snapshot) 1`] = `
<div
  className={undefined}>
  <div>
    <div
      className={undefined}>
      <div
        className={undefined} />
    </div>
  </div>
  <div
    className="Flex"
    style={
      Object {
        "WebkitAlignItems": "center",
        "alignItems": "center",
        "boxSizing": "border-box",
        "display": "flex",
        "msAlignItems": "center",
      }
    }>
    <div
      className="Box"
      style={
        Object {
          "boxSizing": "border-box",
        }
      }>
      <div
        className={undefined}
        size={34}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "borderRadius": "50%",
            "color": "#ffffff",
            "display": "inline-flex",
            "fontSize": 17,
            "height": 34,
            "justifyContent": "center",
            "muiPrepared": true,
            "userSelect": "none",
            "width": 34,
          }
        } />
    </div>
    <div
      className="Box"
      style={
        Object {
          "boxSizing": "border-box",
        }
      }>
      <button
        disabled={false}
        href={undefined}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOut={[Function]}
        onTouchStart={[Function]}
        onTouchTap={[Function]}
        style={
          Object {
            "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
            "background": "none",
            "border": 10,
            "boxSizing": "border-box",
            "cursor": "pointer",
            "display": "inline-block",
            "fontFamily": "Roboto, sans-serif",
            "fontSize": 0,
            "fontWeight": "inherit",
            "height": 48,
            "margin": 0,
            "muiPrepared": true,
            "outline": "none",
            "overflow": "visible",
            "padding": 12,
            "position": "relative",
            "textDecoration": "none",
            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "verticalAlign": null,
            "width": 48,
          }
        }
        tabIndex={0}
        type="button">
        <div
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}>
          <svg
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            style={
              Object {
                "color": "rgba(0, 0, 0, 0.87)",
                "display": "inline-block",
                "fill": "#fff",
                "height": 24,
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "width": 24,
              }
            }
            viewBox="0 0 24 24">
            <path
              d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z" />
          </svg>
        </div>
      </button>
    </div>
  </div>
</div>
`;
